backend/src/activitypub/routes/webfinger.js

```diff
@@ -0,0 +1,59 @@
+import express from 'express'
+import CONFIG from '../../config/'
+import cors from 'cors'
+
+const debug = require('debug')('ea:webfinger')
+const regex = /acct:([a-z0-9_-]*)@([a-z0-9_-]*)/
+
+const createWebFinger = name => {
+  const { host } = new URL(CONFIG.CLIENT_URI)
+  return {
+    subject: `acct:${name}@${host}`,
+    links: [
+      {
+        rel: 'self',
+        type: 'application/activity+json',
+        href: `${CONFIG.CLIENT_URI}/activitypub/users/${name}`,
+      },
+    ],
+  }
+}
+
+export async function handler(req, res) {
+  const { resource = '' } = req.query
+  // eslint-disable-next-line no-unused-vars
```

Comment:
why would this fail lint? you are using `name` and `domain` on line 26??

Reply:
It's the `_` that's being unused.

Reply:
that shouldn't fail linting... it's kind of the purpose to use the `_` in this way